Up2dateService/Up2dateShared/ISettingsManager.cs

```diff
@@ -9,6 +9,8 @@ public interface ISettingsManager
         string RequestCertificateUrl { get; set; }
         string CertificateSerialNumber { get; set; }
 
+        string UpdateVersionMarker { get; set; }
```

Comment:
Not used so far. Remove.
Imtroduce it in the PR where the upgrade issue is being fully addressed